api/app/Models/Article.php

```diff
@@ -75,6 +76,8 @@ class Article extends IndexedModel
         'sections_display' => 'json',
     ];
 
+    protected $indexNested = ['tags', 'articlePermalinks', 'author', 'articleMetas'];
```

Comment:
I think this would make more sense to be labelled `$indexRelations` to indicate what those variables map to.